ansible/inventory/hosts

```diff
@@ -1,2 +1,2 @@
 [dev]
```

Comment:
Dev environment changes such as the Vagrant and Ansible configuration should be merged back to the dev-env branch too.